mne/forward/forward.py

```diff
@@ -1003,7 +1003,8 @@ def _restrict_gain_matrix(G, info):
 
 
 def compute_depth_prior(G, gain_info, is_fixed_ori, exp=0.8, limit=10.0,
-                        patch_areas=None, limit_depth_chs=False):
+                        patch_areas=None, limit_depth_chs=False,
+                        loose_method='svd'):
```

Comment:
need docstring update

Reply:
It doesn't even have a docstring. Do you want me to add a complete one for it (and compute_depth_prior)?